boxes/templates/boxes/box_create.html

```diff
@@ -19,7 +19,7 @@ <h2 class="text-dark">Create New Box</h2>
                 </p>
                 <p>
                     {{ form.description.errors }}
-                    {{ form.description.label_tag }}
+                    {{ form.descyription.label_tag }}
```

Comment:
btw, o meu github for mac tem as cores trocadas do que aparentemente vocês costumam ter, não sei se afetará tb aqui no site.